client/app/components/sections/ManageAvailability/ManageAvailability.js

```diff
@@ -70,6 +81,15 @@ class ManageAvailability extends Component {
     loadInitialDataIfNeeded(this.props);
   }
 
+  componentWillUpdate(nextProps) {
+    // manage location hash
+    if (nextProps.isOpen && window.location.hash !== '#manage-availability') {
```

Comment:
I don't think the `componentWillUpdate` lifecycle method will be run in the server. That method is called when the component is rendered with new props, and the server only renders the component once.

FYI, we are not server rendering this app at all, but that shouldn't affect how we do the components within this app.